tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -234,8 +234,8 @@ strategy = ""
 # This parameter is introduced in v7.1.0.
 # region-check-backoff-limit = 1800
 
-# In Physical Import Mode, this parameter controls the I/O block size for the local sorting file. You can increase this value when IOPS is a bottleneck.
-# This parameter is introduced in v7.6.0. The default value is "16KiB".
+# In Physical Import Mode, this parameter controls the I/O block size for sorting local files. When the disk IOPS is a bottleneck, you can increase this value to improve data import performance.
+# This parameter is introduced in v7.6.0. The default value is "16KiB". The value must be greater than or equal to `1B`. Note that if you only specify a number (for example, `16`), the unit is Byte instead of KiB.
```

Comment:
```suggestion
# This parameter is introduced in v7.6.0. The default value is "16KiB". The value must be of an `INTEGER` type, and must be greater than or equal to `1B`. Note that if you only specify a number (for example, `16`), the unit is Byte instead of KiB.
```